### PR TITLE
Elevation enhancements

### PR DESCRIPTION
**Pull request by landonreed**
Enhances the elevation profile chart  according to the changes outlined in conveyal/trimet-mod-otp#171. Here's how the new chart appears:

image

### CODE REVIEW

File: lib/components/map/leg-diagram.js [hunk 1]
```diff
@@ -130,84 +155,153 @@ class LegDiagram extends Component {
...
+            y={y - 10}
+            fontSize={11}
+            textAnchor='middle'>
+            {this._formatElevation(elevConverted)}
+          </text>
+        )
+      }
     }
-
+    // console.log({ traversed, width, xAxisCompression })
```
Comment (evansiroky) [inline lib/components/map/leg-diagram.js]: Still needed?

Comment (landonreed) [reply lib/components/map/leg-diagram.js]: Nah. Will remove.

File: lib/components/map/base-map.js [hunk 2]
```diff
@@ -248,8 +248,8 @@ class BaseMap extends Component {
 
   render () {
     const { config, children, diagramLeg, elevationPoint, popupLocation } = this.props
-    const { baseLayers } = this.props.config.map
-
+    const { baseLayers } = config.map
+    const showElevationProfile = Boolean(config.elevationProfile)
```
Comment (evansiroky) [inline lib/components/map/base-map.js]: Can you make a PR to https://github.com/conveyal/trimet-mod-otp/ with this added config setting enabled?

Comment (landonreed) [reply lib/components/map/base-map.js]: https://github.com/conveyal/trimet-mod-otp/pull/186

**Comment by landonreed** [comment]
Just pushed another commit that accounts for the move of a bunch of files from the TriMet repo to this one.

**Pull request merged**